lib/LedgerSMB/Upgrade_Tests.pm

```diff
@@ -150,15 +161,16 @@ Repair query key to set the values if we can repair
 
 =cut
 
-has id_where => (is => 'ro', isa => 'Str', required => 0, default => 'id');
+has id_where => (is => 'ro', isa => 'Str', required => 0, default => '');
 
-=item id_column
+=item id_columns
```

Comment:
You haven't changed all references to `id_column` to the new name (there's one left, as far as I can tell)

Reply:
Fixed